Add basic explanation to ProgrammableAndroids section. (#106)
Slimefun/Wiki: pages/Androids.md

> Programmable Androids category contain robots for the user to automate some task.

Reviewer: ```suggestion
The Programmable Androids category contains robots that are able to automate tasks.
```

> Programmable Android category contains tools for the user for automating their task to make their life easier.

Author: ```suggestion
Programmable Androids category contain robots for the user to automate some task.
```
Like this?
